tests/e2e/swarm_test.go

```diff
@@ -1,144 +0,0 @@
-// 12. Verify the volume is in attached status
-// 13. Delete the volume - expect fail
-// 14. Remove the service
-// 15. Verify the service is gone
-// 16. Verify the volume is in detached status
-func (s *SwarmTestSuite) TestDockerSwarm(c *C) {
-	log.Printf("START: swarm_test.TestDockerSwarm")
-
-	fullVolumeName := verification.GetFullVolumeName(s.master, s.volumeName)
-	opts := "--replicas 1 --mount type=volume,source=" + fullVolumeName + ",target=/vol,volume-driver=" + constant.VDVSPluginName + constant.TestContainer
-	out, err := dockercli.CreateService(s.master, s.serviceName, opts)
-	c.Assert(err, IsNil, Commentf(out))
-
-	status := verification.IsDockerServiceRunning(s.master, s.serviceName, 1)
-	c.Assert(status, Equals, true, Commentf("Service %s is not running", s.serviceName))
-
-	status, _ = verification.IsDockerContainerRunning(s.swarmNodes, s.serviceName, 1)
-	c.Assert(status, Equals, true, Commentf("Container %s is not running", s.serviceName))
-
-	status = verification.VerifyAttachedStatus(s.volumeName, s.master, s.esxName)
-	c.Assert(status, Equals, true, Commentf("Volume %s is not attached", s.volumeName))
-
-	out, err = dockercli.ScaleService(s.master, s.serviceName, 2)
-	c.Assert(err, IsNil, Commentf(out))
-
-	status = verification.IsDockerServiceRunning(s.master, s.serviceName, 2)
-	c.Assert(status, Equals, true, Commentf("Service %s is not running", s.serviceName))
-
-	status, host := verification.IsDockerContainerRunning(s.swarmNodes, s.serviceName, 2)
-	c.Assert(status, Equals, true, Commentf("Container %s is not running on any hosts", s.serviceName))
-
-	containerName, err := dockercli.GetContainerName(host, s.serviceName+".1")
-	c.Assert(err, IsNil, Commentf("Failed to retrieve container name: %s", containerName))
-	out, err = dockercli.StopService(host, containerName)
-	c.Assert(err, IsNil, Commentf(out))
-
-	status = verification.IsDockerServiceRunning(s.master, s.serviceName, 2)
-	c.Assert(status, Equals, true, Commentf("Service %s is not running", s.serviceName))
-
-	status, _ = verification.IsDockerContainerRunning(s.swarmNodes, s.serviceName, 2)
-	c.Assert(status, Equals, true, Commentf("Container %s is not running", s.serviceName))
-
-	status = verification.VerifyAttachedStatus(s.volumeName, s.master, s.esxName)
-	c.Assert(status, Equals, true, Commentf("Volume %s is not attached", s.volumeName))
-
-	out, err = dockercli.DeleteVolume(s.master, s.volumeName)
-	c.Assert(err, NotNil, Commentf("Expected error does not happen"))
-
-	out, err = dockercli.RemoveService(s.master, s.serviceName)
-	c.Assert(err, IsNil, Commentf(out))
-
-	out, err = dockercli.ListService(s.master, s.serviceName)
-	c.Assert(err, NotNil, Commentf("Expected error does not happen"))
-
-	status = verification.VerifyDetachedStatus(s.volumeName, s.master, s.esxName)
-	c.Assert(status, Equals, true, Commentf("Volume %s is still attached", s.volumeName))
-
-	log.Printf("END: swarm_test.TestDockerSwarm")
-}
```

Comment:
I am assuming deletion of this file was by mistake. If not; why? //CC @shaominchen